train/t0/tr11f-6B3-ml-t0.slurm

```diff
@@ -80,7 +79,6 @@ OPTIMIZER_ARGS=" \
     --adam-eps 1e-8 \
     --lr 1e-3 \
     --lr-decay-style constant \
-    --lr-warmup-samples $LR_WARMUP_SAMPLES \
```

Comment:
We used `Adafactor` ... so technically I don't know what parameters matter (typically we used a `decay` argument, which I don't know how it translates to `Adam` optimizer)